addons/armaos/functions/fnc_terminal_uiOnTex_addUpdateAllEventHandler.sqf

```diff
@@ -0,0 +1,54 @@
+/**
+ * Adds an 5 Sec Per-Frame-Event-Handler for the "UI on texture" feature. This will update all contents regularly.
+ *
+ * Arguments:
+ * 1: Computer <OBJECT>
+ * 2: Console <DIALOG>
+ *
+ * Results:
+ * None
+ */
+
+params ["_computer", "_consoleDialog"];
+
+private _updateInterval = 5;
```

Comment:
Why do we need to update regularly? Isn't everything pushed when it changes?

Reply:
Yes, everything is pushed, but if player a uses the Terminal and anly reads it without any interaction, there is nothing to push. If player b enters the circle, he won't see the actual content/interface. Imagine a player is not sitting in front of his real life computer and blocking the laptop, or thinking very long about the next command or perhaps unconciousness. That's why there will be an auto-push-interval.